test/visual/default.html

```diff
@@ -41,6 +41,13 @@
     </vaadin-checkbox-group>
   </div>
 
+  <div id="theme-vertical-group-tests">
+    <vaadin-checkbox-group label="Choose the option" theme="vertical">
```

Comment:
nit: Why does this one have a different label than the other samples ("Choose a number" vs "Choose the option")?

Reply:
I changed it to "Choose a number".